.github/workflows/gradle.yml

```diff
@@ -495,6 +510,15 @@ jobs:
       - name: Automatic Migration Acceptance Test
         run: SUB_BUILD=PLATFORM ./gradlew :airbyte-tests:automaticMigrationAcceptanceTest --scan -i
 
+      - name: Generate Test Report
+        uses: dorny/test-reporter@v1
+        if: always()    # run this step even if previous step failed
+        with:
+          name: Platform Test Report with Docker E2E Test
```

Comment:
I'm really trying to take to heart the idea that we strive to communicate as clearly and loudly as possible when showing developers how to fix their own problems.  

is there a name for this test that more clearly screams what it is?  How can we get every developer to know there is a better error message process over here?